### ISSUE TITLE
[Bug] SwipeView broken in 4.5.0.356; creating huge number of overlapping SwipeItemViews on reveal

### ISSUE THREAD

**Issue by tomeverin** [xamarin/Xamarin.Forms#9735]
### Description
When swiping to reveal right items, UI frame rate of SwipeView degrades. Frames with shadows reveal that for each pixel scrolled by the SwipeView.Content another set of SwipeItems appears to be added to the visual tree. These persist after hiding the swipe items again, and performance is cumulatively degraded with subsequent interactions with SwipeView.

---

**This behaviour breaks SwipeView + CollectionView**
New in Xamarin.Forms 4.5.0.356
Not reproducible when downgrading to Xamarin.Forms 4.4.0.991757
Only tested on Xamarin.Android + Android Emulator; see below for GIFs

---

### Steps to Reproduce

1. Create a new Xamarin android app project using usual VS template for to-do list (Master Detail)

2. Populate data in ViewModel
```
 public class ItemsViewModel : BaseViewModel
    {
        public List<Item> Items { get; set; }
        public Command LoadItemsCommand { get; set; }

        public ItemsViewModel()
        {
            List<Item> curList = new List<Item>();
            curList.Add(new Item() { Text = "Item 1" });
            curList.Add(new Item() { Text = "Item 2" });
            curList.Add(new Item() { Text = "Item 3" });
            curList.Add(new Item() { Text = "Item 4" });
            curList.Add(new Item() { Text = "Item 5" });
            curList.Add(new Item() { Text = "Item 6" });

            Items = curList;
        }
...
```

3. Adjust the `ItemPage.xaml` XAML
```
...
<StackLayout>
        <CollectionView IsGrouped="false" ItemsSource="{Binding Items}" BackgroundColor="White">

            <CollectionView.ItemTemplate>
                <DataTemplate>
                    <SwipeView>

                        <SwipeView.RightItems>
                            <SwipeItems Mode="Reveal">
                                <SwipeItemView>
                                    <StackLayout Margin="10, 20, 10, 20">
                                    <Frame HasShadow="True" CornerRadius="30">
                                        <BoxView HeightRequest="45" WidthRequest="30" BackgroundColor="LightBlue"></BoxView>
                                    </Frame>
                                    </StackLayout>
                                </SwipeItemView>
                                <SwipeItemView>
                                    <StackLayout Margin="10, 20, 10, 20">
                                        <Frame HasShadow="True" Margin="10, 20, 10, 20"  CornerRadius="30">
                                        <BoxView HeightRequest="45"  WidthRequest="30" BackgroundColor="LightGoldenrodYellow"></BoxView>
                                    </Frame>
                                    </StackLayout>

                                </SwipeItemView>

                            </SwipeItems>
                            
                        </SwipeView.RightItems>
                        <SwipeView.Content>
                            <Grid>
                                <Grid.RowDefinitions>
                                    <RowDefinition Height="120"></RowDefinition>
                                </Grid.RowDefinitions>

                                <Frame Grid.Row="0" HasShadow="True" Margin="10,6,10,6" Padding="3">

                                    <StackLayout>
                                        <Label Text="{Binding Text}" FontSize="Medium" Margin="0"></Label>
                                    </StackLayout>


                                </Frame>
                            </Grid>
                        </SwipeView.Content>

                    </SwipeView>
                    
                </DataTemplate>
            </CollectionView.ItemTemplate>
        </CollectionView>
    </StackLayout>
...
```
4. Add experimental flag for SwipeView to `MainActivity.cs` in the Android project
```
global::Xamarin.Forms.Forms.SetFlags(new string[] { "SwipeView_Experimental" });
```
5. Make sure the Xamarin.Forms version is 4.4.0.991757

6. Run the app in Android Emulator; you will not observe any problem when interacting with SwipeView

7. Upgrade to Xamarin.Forms 4.5.0.356

8. Run the app in the Android Emulator; you will see performance of Swipe View frame rate degrade as you interact with it more. Additionally, the shadow around the SwipeItemView Frames will darken, suggesting overlay of multiple copies of the views.

---> see below for GIFs

### Expected Behavior
See below for GIF

### Actual Behavior
See below for GIF

### Basic Information

- Version with issue: Xamarin.Forms 4.5.0.356
- Last known good version: Xamarin.Forms 4.4.0.991757
- IDE: Visual Studio Community 16.4.5

### Screenshots
Forms 4.4.0.991757:
![forms4 4 0 991757](https://user-images.githubusercontent.com/24466690/75412398-3d6e3d00-5972-11ea-92a6-73bb49b6a1af.gif)

Forms 4.5.0.356:
![forms4 5 0 356](https://user-images.githubusercontent.com/24466690/75412428-54149400-5972-11ea-82d4-ac6d96eb9c2e.gif)

**Comment by jsuarezruiz** [comment]
@tomeverin Thanks for the detailed feedback. It is a regression in 4.5 that will be fixed by https://github.com/xamarin/Xamarin.Forms/pull/9725
I have added your example to the samples to validate the issue. The only change I have added is transparency to the SwipeView content to see when the SwipeItems are created and when are disposed.

![issue9735](https://user-images.githubusercontent.com/6755973/75438197-d6438f00-5957-11ea-912d-97939bdf1357.gif)
